### ISSUE TITLE
CC versions are no longer stored in the cache file

### ISSUE THREAD

**Issue by AlCalzone** [zwave-js/zwave-js#799]
all endpoints show version 0.
```json
                "0x8e": {
                    "name": "Multi Channel Association",
                    "endpoints": {
                        "0": {
                            "isSupported": true,
                            "isControlled": false,
                            "version": 0
                        },
                        "1": {
                            "isSupported": true,
                            "isControlled": false,
                            "version": 0
                        },
                        "2": {
                            "isSupported": true,
                            "isControlled": false,
                            "version": 0
                        }
                    }
                },
```

**Comment by AlCalzone** [comment]
Locally it works - maybe this broke when converting the DB files?

**Comment by AlCalzone** [comment]
fixed in 2c536e1
